src/js/dataMapping/contracts/transactionTable.js

```diff
@@ -5,33 +5,38 @@
 
 const tableSearchFields = {
     columnWidths: {
-        modification: 230,
-        action_date: 200,
-        amount: 220,
-        description: 380
+        modification: 184,
```

Comment:
Is there a reason the column widths were reduced? Having the columns too narrow will cause the header cells to wrap (which don't work correctly) when alternative fallbacks fonts are loaded (government computers are configured to ignore custom fonts). Can we restore the wider column widths?